### ISSUE TITLE
Position Class missing types

### ISSUE THREAD

**Issue by CaitieBoeyens** [kamranahmedse/driver.js#139]
In the constructor for Position I get the following error when using  Driver in a Typescript project

`613:21 Binding element 'left' implicitly has an 'any' type.
 613:21 Binding element 'top' implicitly has an 'any' type. 
613:21 Binding element 'bottom' implicitly has an 'any' type. 
613:21 Binding element 'right' implicitly has an 'any' type.`

**Comment by schmulschubiak** [comment]
Maybe could be easy to fix? In types/index.d.ts in Line 615 just add any-Type?
Instead of 
```class Position {
      constructor({
                    left,
                    top,
                    bottom,
                    right,
                  });
```
use
```class Position {
      constructor({
                    left,
                    top,
                    bottom,
                    right,
                  }: any);
```

**Comment by Attrash-Islam** [comment]
Can you produce this in CodeSandbox? I'd be happy to patch a PR

**Comment by schmulschubiak** [comment]
> Can you produce this in CodeSandbox? I'd be happy to patch a PR
Ah good to hear! I think, CodeSandbox does'nt support tslint right now, so I correct typing in my own project to test it. 
Could I just make/send a PR to you?

**Comment by Attrash-Islam** [comment]
Aaaah, I see your point. I think that the typings were written in not strict mode but was referenced in strict tsconfig.json which shows the errors. Adding types to those things should be a PR for fixes. I hope you already opened a PR 🎉

**Comment by kamranahmedse** [comment]
This has been fixed in the latest release (0.9.0). Thanks guys!
